doc/developers/advanced_installation.rst

```diff
@@ -87,7 +87,7 @@ feature, code or documentation improvement).
 
      python3 -m venv sklearn-env
      source sklearn-env/bin/activate
-     pip install wheel numpy scipy cython
+     pip install wheel numpy scipy cython meson-python
```

Comment:
Is ninja also installed automatically with pip? It wasn't the case in the past but I haven't tried again recently.

Reply:
Indeed you need `pip install meson-python ninja`. My understanding is that the system `ninja` can be found if it exists, which may be preferred by Meson people. We prefer that everything is contained in the Python environment for convenience so +1 for `pip install ninja meson-python`.

Full disclosure with conda, `conda install meson-python` will install `ninja`, but I I am fine doing `conda install meson-python ninja` anyway for consistence in the instructions between `conda` and `pip`